nss/Survey_NSS/02_Table of Means NSS 70.do

```diff
@@ -51,7 +52,7 @@ gen du_share = durable_other/asset
 gen gl_share = gold / asset
 
 sum re_share du_share gl_share
-sum re_share du_share gl_share [aw = pwgt] //gold is 11% consistent with paper. 
+sum re_share du_share gl_share [aw = hhwgt] //gold is 11% consistent with paper. 
```

Comment:
pp15: "In Panel A, we compute averages across households using population weights, as indicated in each survey"

Reply:
Nadeem clarified that the population weight might be defined as household weight in the paper.